audit.sh

```diff
@@ -20,4 +20,5 @@ sleep 5400
 echo "Generating reports"
 
 cd WorkerVersions
+find . -type f -not -name "workers.json" -delete # remove everything except workers.json
```

Comment:
Good question.. So it was added to simplify parsing and building of the published page with history

It works like this:
with each report we dump state into a `workers.json` file with all pools and implementations 
we commit this file

then `build-docs-history.sh` is going through git revisions to collect all those files and create a time series 

but now as I'm writing this I realize we can simply delete it, it will be recreated, and it only contains the information about the current report, so it will be regenerated anyways :+1: